### DISCUSSION TITLE
Qnap - Missing Files

### DISCUSSION THREAD

**Discussion by jvward** [haugene/docker-transmission-openvpn#2526] [Q&A]
I am tyring to get this docker running on QNAP, and its downloading and seeding, but the files are missing from file station no matter what I do.  Here is my config:

version: '3.3'
services:
    transmission-openvpn:
        cap_add:
            - NET_ADMIN
        volumes:
            - '/share/CACHEDEV1_DATA/DataVol1/Storage/Transmission/data/:/data'
            - '/share/CACHEDEV1_DATA/DataVol1/Storage/Transmission/config/:/config'

        environment:
            - OPENVPN_PROVIDER=PIA
            - OPENVPN_CONFIG=france
            - OPENVPN_USERNAME=yyyy
            - OPENVPN_PASSWORD=xxxxx
            - LOCAL_NETWORK=192.168.1.0/24
        logging:
            driver: json-file
            options:
                max-size: 10m
        ports:
            - '9091:9091'
        image: haugene/transmission-openvpn

I have used a torrent VPN tester and its working.  I can download larger files and they download.  I can restart the container and they will continue to seed, but when I check the mapped folders no matter what the files are missing. Any ideas how I can fix it would be greatly appreciated, and for the life of my I cant find them anywhere else on the file system.  I tried building the docker in container station natively and also I tried setting it up in portainer.  Both have the same issue.

**Comment by pkishino** [comment]
is there anything at /share/CACHEDEV1_DATA/DataVol1/Storage/Transmission/data/ 
? check via the cli.. no idea how QNAP works, best bet would be to check some snap forums..possibly the location you mapped isn't visible in file station
